### PR TITLE
docs: suggestions for updates to cluster docs

### PR DESCRIPTION
**Pull request by seialkali**
## Jira Ticket

https://jira.d2iq.com/browse/D2IQ-86815

## Description of changes being made

These are just some things I noticed when reading through the Cluster docs, which I wanted to suggest changes for. However, I'm happy to revert anything that isn't quite working, or include any additional changes.

### Preview

You can preview the docs build using the following URL, adding the PR # where specified:
http://docs-d2iq-com-pr-4253.s3-website-us-west-2.amazonaws.com/

## Checklist

- [ ] Test all commands and procedures, if applicable.
- [ ] Update all links if you are moving a page.
- [ ] Add release date to Release Notes page in the following format: <Package> was released on <Day>, <Month> <Year> Example: `Mesosphere® DC/OS™ 2.1.0 was released on 9, June 2020`

See the [contribution guidelines](https://github.com/mesosphere/dcos-docs-site/blob/main/CONTRIBUTING.md) for more information.

### CODE REVIEW

File: pages/dkp/kommander/2.2/clusters/attach-cluster/cluster-no-network-restrictions/index.md [hunk 1]
```diff
@@ -11,18 +11,20 @@ beta: false
...
+1. The **Cluster Name** field will automatically populate with the name of the cluster is in the kubeconfig. You can edit this field with the name you want for your cluster.
 
-1. Add labels to classify your cluster as needed.
+1. The **Context** select list is populated from the kubeconfig. Select the desired context with admin privileges from the **Context** select list.
 
-1. Select the **Submit** button to begin the cluster attachment process.
+1. Add labels to classify your cluster as needed.
 
-![Add Cluster Connect](../../../img/add-cluster-connect.png)
+1. Select **Submit** to attach your cluster.
```
Comment (vacas5) [inline pages/dkp/kommander/2.2/clusters/attach-cluster/cluster-no-network-restrictions/index.md:30]: Just FYI there are some notes in the Contributing doc about using includes if you have the exact same content in multiple places. For future reference though. I usually abstract stuff out based on the rule of threes, and I only see this duplicated once.

**Review by vacas5 — APPROVED** [review]

File: pages/dkp/kommander/2.2/clusters/attach-cluster/attach-eks-cluster/index.md [hunk 2]
```diff
@@ -116,19 +116,21 @@ This procedure requires the following items and configurations:
...
+1.  Select **Attach Cluster**.
 
-1.  Paste the contents of your clipboard (or upload the file you created) into the Kubeconfig File text box.
+1.  Select the **No additional networking restrictions** card.
 
-1.  Assign a name and add any desired labels for the cluster.
+1.  The **Cluster Configuration** section of the form accepts a kubeconfig file that you can paste, or upload, into the field. Paste the contents of your clipboard (or upload the file you created) into the kubeconfig field.
 
-1.  Confirm you are assigning the cluster to your desired workspace.
+1.  The **Cluster Name** field will automatically populate with the name of the cluster in the kubeconfig. You can edit this field with the name you want for your cluster.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/clusters/attach-cluster/attach-eks-cluster/index.md:129]:
```suggestion
1.  The **Cluster Name** field automatically populates with the name of the cluster in the kubeconfig. You can edit this field with the name you want for your cluster.
```

File: pages/dkp/kommander/2.2/clusters/attach-cluster/cluster-with-network-restrictions/index.md [hunk 3]
```diff
@@ -13,19 +13,21 @@ Use this option when you want to attach a cluster that is in a DMZ, behind a NAT
...
-<!--- ![Add Cluster Networking Options](/dkp/kommander/1.4/img/cluster-has-networking-restrictions.png) --->
+    <!--- ![Add Cluster Networking Options](/dkp/kommander/1.4/img/cluster-has-networking-restrictions.png) --->
 
-1. Enter the **Cluster Name** of the cluster you're attaching and select a **Workspace** from the dropdown list (if entering the **Add Cluster** menu from the Global workspace).
+1. Enter the **Cluster Name** of the cluster you're attaching.
 
 1. Create additional new Labels as needed.
 
-1. Select the **Load Balancer Hostname** which is the Ingress for the cluster from the dropdown menu. You will want the hostname to match the Kommander Host cluster that you are attaching your existing cluster with network restrictions to.
+1. Select the hostname that is Ingress for the cluster from the **Load Balancer Hostname** dropdown menu. You will want the hostname to match the Kommander Host cluster that you are attaching your existing cluster with network restrictions to.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/clusters/attach-cluster/cluster-with-network-restrictions/index.md:30]:
```suggestion
1. Select the hostname that is the Ingress for the cluster from the **Load Balancer Hostname** dropdown menu. The hostname must match the Kommander Host cluster to which you are attaching your existing cluster with network restrictions.
```

File: pages/dkp/kommander/2.2/clusters/creating-konvoy-cluster-advanced/index.md [hunk 4]
```diff
@@ -19,12 +19,14 @@ Depending on your infrastructure, Konvoy can generate a set of cluster objects t
...
 
-Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, taking time for the cluster to be ready and fully deploy its components. The cluster will retry to join automatically and should resolve once it is fully provisioned.
+    - **Workspace**: The workspace where this cluster belongs (if within the Global workspace).
+    - **Cluster YAML**: Paste or upload your customized set of cluster objects into this field. Only valid YAML will be accepted.
+    - **Add Labels**: By default, your cluster has labels that reflect the infrastructure provider provisioning. For example, your AWS cluster may have a label for the data center region and `provider: aws`. Cluster labels are matched to the selectors created for [projects][projects]. Changing a cluster label may add or remove the cluster from projects.
+
+1. Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, taking time for the cluster to be ready and fully deploy its components. The cluster will retry to join automatically and should resolve once it is fully provisioned.
```
Comment (JohnB64-JBV) [inline pages/dkp/kommander/2.2/clusters/creating-konvoy-cluster-advanced/index.md:30]:
```suggestion
1. Click **Create** to begin provisioning the Konvoy cluster. This step may take a few minutes, taking time for the cluster to be ready and fully deploy its components. The cluster automatically tries to join, and should resolve after it is fully-provisioned.
```

**Review by JohnB64-JBV — CHANGES_REQUESTED** [review]

File: pages/dkp/kommander/2.2/clusters/applications/index.md [hunk 5]
```diff
@@ -8,17 +8,15 @@ excerpt:
...
 
 ## Applications
 
-You can visit the "Applications" section of the cluster's detail page to view the workspace applications enabled for the cluster. Applications are displayed in groups with all similar enabled applications.
+Under the **Applications** section of the cluster's detail page, you can view the workspace applications enabled for the cluster, grouped by category.
 
 ![Cluster Detail Page - Applications](../../img/cluster-detail-page-applications.png)
 
-Under the "Applications" section, you can view the current status of the enabled applications on the cluster. The status is displayed on the application card. Hovering on the status displays details about the status of the application.
+In this section, you can also view the current status of the enabled applications on the cluster. The status is displayed on the application card. Hovering on the status displays details about the status of the application.
```
Comment (Colin1841) [inline pages/dkp/kommander/2.2/clusters/applications/index.md:19]:
```suggestion
In this section, you can also view the current status of the enabled applications on the cluster, on each application card. Hovering on the status displays details about the status of the application.
```

**Review by Colin1841 — APPROVED** [review]

**Review by JohnB64-JBV — APPROVED** [review]

**Pull request merged**